CMSgov/design-system
# [WNMGDS-2191] Clearly differentiate between doc headings and example headings

## Summary

https://jira.cms.gov/browse/WNMGDS-2191

The accessibility violation that spawned this work was that some documentation pages had multiple `h1` headings. The headings in the embedded examples were intermingled with the headings for the document itself. This could result in a confusing screen-reader experience. We explored several alternatives to this change, including https://github.com/CMSgov/design-system/pull/2386 and even removing the embedded examples entirely and linking to Storybook instead. Given our technical audience, we've landed on a content-only solution, which is to clearly label the headings that are examples.

## How to test

[Demo of typography/content page](https://cmsgov.github.io/design-system/branch/pwolfert/doc-example-headings/foundation/typography/content/)
[Demo of typography/headings page](https://cmsgov.github.io/design-system/branch/pwolfert/doc-example-headings/foundation/typography/headings/)

Try the headings page in the medicare theme to see alternate content.

## Checklist




### If this is a change to code:


### If this is a change to documentation:
